src/main/webapp/js/datahandlers/collegepagedatahandler.js

```diff
@@ -12,96 +12,122 @@
-      netCostForFifthQuintile: json["netCostForFifthQuintile"],
-      cumulativeMedianDebt: json["cumulativeMedianDebt"]
-    };
+  if (json !== undefined) {
+    return /** @type {!CollegeObject} */ ({
+      schoolName: json['schoolName'],
+      institutionType: json['institutionType'],
+      acceptanceRate: json['acceptanceRate'],
+      averageACTScore: json['averageACTScore'],
+      totalCostAttendance: json['totalCostAttendance'],
+      netCostForFirstQuintile: json['netCostForFirstQuintile'],
+      netCostForSecondQuintile: json['netCostForSecondQuintile'],
+      netCostForThirdQuintile: json['netCostForThirdQuintile'],
+      netCostForFourthQuintile: json['netCostForFourthQuintile'],
+      netCostForFifthQuintile: json['netCostForFifthQuintile'],
+      cumulativeMedianDebt: json['cumulativeMedianDebt']
+    });
   } else {
     throw new Error(ERROR_MESSAGE_JSON_UNDEFINED);
   }
 };
 
 /**
  * Fetch request to the data servlet and return the JSON response.
- * @return {*} - The JSON response.
+ * @return {!Promise<!CollegeObject|undefined>} - The JSON response.
  * @private
  */
 const loadCollegeJson_ = async () => {
   const response = await fetch(COLLEGE_SERVLET_ENDPOINT);
   if (response.ok) {
-    return await response.json();
+    return /** @type {!CollegeObject|undefined} */ (await response.json());
   } else {
     throw new Error(`${response.statusText}. Status: ${response.status}.`);
   }
 };
 
 /**
  * Render the soy template's html by attaching it to the desired DOM element.
- * @param {Element} element - The DOM element where the template should be rendered to.
+ * @param {!Element} element - The DOM element where the template should be rendered to.
  */
 const loadCollegeData = async (element) => {
   let json = undefined;
 
   try {
     json = await loadCollegeJson_();
-  } catch(err) {
-    loadErrorPage_(element, OCCURRENCE_JSON_UNDEFINED, ACTION_JSON_UNDEFINED, err);
+  } catch (error) {
+    loadErrorPage_(
+        element, OCCURRENCE_JSON_UNDEFINED, ACTION_JSON_UNDEFINED, error);
   }
 
   try {
     const data = await convertFromJsonToTemplate_(json);
     const html = collegepage(data);
     element.innerHTML = html;
-  } catch(err) {
-    loadErrorPage_(element, OCCURRENCE_SERVER_ERROR, ACTION_SERVER_ERROR, err);
-  }  
-}
+  } catch (error) {
+    loadErrorPage_(
+        element, OCCURRENCE_SERVER_ERROR, ACTION_SERVER_ERROR, error);
+  }
+};
 
 /**
  * Loads the error page.
- * @param 
  * @private
  */
-const loadErrorPage_ = (element,  occurrence, action, error) => { 
-  const errorPage = new ErrorPageView();
-  errorPage.renderErrorPage(element, occurrence, action, error.toString());
-}
+const loadErrorPage_ = (element, occurrence, action, error) => {
+  //  const errorPage = new ErrorPageView(/** @type {!ErrorData|undefined} */ ({
+  //    occurrence: occurrence,
+  //    action: action,
+  //    errorMessage: error.toString()
+  //  }));
+  //  errorPage.renderView(element);
```

Comment:
Remove.

Reply:
This looks bad because it appears as if it's swallowing the exceptions, but this file is not being used at the moment.